--- origsrc/xfce4-cpugraph-plugin-1.0.1/panel-plugin/os.c	2010-11-07 06:58:22.000000000 -0600
+++ src/xfce4-cpugraph-plugin-1.0.1/panel-plugin/os.c	2011-02-27 23:54:19.608998100 -0600
@@ -31,7 +31,7 @@
 #include <stdio.h>
 #include <string.h>
 
-#if defined (__linux__) || defined (__FreeBSD_kernel__)
+#if defined (__linux__) || defined (__FreeBSD_kernel__) || defined (__CYGWIN__)
 #define PROC_STAT "/proc/stat"
 #define PROCMAXLNLEN 256 /* should make it */
 #endif
@@ -67,7 +67,7 @@
 static kstat_ctl_t *kc;
 #endif
 
-#if defined (__linux__) || defined (__FreeBSD_kernel__)
+#if defined (__linux__) || defined (__FreeBSD_kernel__) || defined (__CYGWIN__)
 guint detect_cpu_number()
 {
 	guint nb_lines= 0;
